BusyList/TaskItem.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text;
+
+namespace BusyList
+{
+    public class TaskItem
+    {
+        public TaskItem(int id, string description, TaskStatus status = TaskStatus.NotStarted)
+        {
+            Id = id;
+            Description = description;
+            TaskStatus = status;
+        }
+        public int Id { get; }
```

Comment:
If we implement relative ids won't this be changing everytime.

Reply:
Yes, but we have not decided on relative IDs and even if we did, that was a post 1.0 feature.